<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsForm</class>
 <widget class="QDialog" name="SettingsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>272</width>
    <height>519</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Настройки</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/icons/settings.png</normaloff>:/icons/icons/settings.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">
QWidget
{
	background-color: #212630;
	color: #CFD2DC;
	border-color: #1C2227;

}

QFrame
{
	background-color: #212630;
	border: 1px solid #212630;
	border-radius: 10px;
}


/*-----QComboBox-----*/
QComboBox
{
    background-color: #232935;
    border: 2px solid;
    border-radius: 15px;
	border-color: #3BB0D7;
    padding-left: 6px;
    color: #CFD2DC;
    height: 20px;

}


QComboBox::disabled
{
	background-color: #232935;
	color: #CFD2DC;
	border-color: #051a39;

}


QComboBox:hover
{
    background-color: #353C4F;

}


QComboBox:on
{
    background-color: #3BB0D7;

}


QComboBox QAbstractItemView
{
    background-color: #1C2227;
    color: #CFD2DC;
    border: 1px solid black;
    selection-background-color: #3BB0D7;
    selection-color: #CFD2DC;
    outline: 0;

}


QComboBox::drop-down
{
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 15px;
    border-left-width: 0px;
    border-left-color: darkgray;
    border-left-style: solid; 
    border-top-right-radius: 3px; 
    border-bottom-right-radius: 3px;

}


QComboBox::down-arrow
{
    image: url(:/icons/icons/arrow-down.png);
    width: 8px;
    height: 8px;
}

/*-----QPushButton-----*/
QPushButton
{
	background-color: #232935;
	color: #CFD2DC;
	font-weight: bold;
	border-style: solid;
	border-width: 2px;
	border-radius: 15px;
	border-color: #3BB0D7;
	padding: 10px;
	highlited:true;
}


QPushButton::hover
{
	background-color: #353C4F;
	color: #CFD2DC;

}


QPushButton::pressed
{
	background-color: #3BB0D7;
	color: #CFD2DC;

}

/*-----QSpinBox &amp; QDoubleSpinBox &amp; QDateTimeEdit-----*/
QSpinBox, 
QDoubleSpinBox,
QDateTimeEdit
{
	background-color: #232935;
	color: #CFD2DC;
	border: 2px solid #3BB0D7;
	border-radius: 10px;
	padding : 2px;

}



QSpinBox:hover, 
QDoubleSpinBox::hover,
QDateTimeEdit::hover
{
    background-color: #232935;
    border: 2px solid #6CB2FF;
    color:  #CFD2DC;
    padding: 2px

}


QSpinBox::up-button,
QDoubleSpinBox::up-button,
QDateTimeEdit::up-button
{
    background-color: #3BB0D7;
	border-top-right-radius: 5px;
	border-top-left-radius: 1px;

}

QSpinBox::down-button,
QDoubleSpinBox::down-button,
QDateTimeEdit::down-button
{
    background-color: #3BB0D7;
	border-bottom-right-radius: 5px;
	border-bottom-left-radius: 1px;

}

QSpinBox::up-button:hover, QSpinBox::down-button:hover,
QDoubleSpinBox::up-button:hover, QDoubleSpinBox::down-button:hover,
QDateTimeEdit::up-button:hover, QDateTimeEdit::down-button:hover
{
    background-color: #45CDFA;
    border: 2px solid #45CDFA;

}

QSpinBox::up-button:pressed, QSpinBox::down-button:pressed,
QDoubleSpinBox::up-button:pressed, QDoubleSpinBox::down-button::pressed,
QDateTimeEdit::up-button:pressed, QDateTimeEdit::down-button::pressed
{
    background-color: #CFD2DC;
    border: 2px solid #CFD2DC;

}

QSpinBox::down-arrow,
QDoubleSpinBox::down-arrow,
QDateTimeEdit::down-arrow

{
    image: url(:/icons/icons/arrow-down.png);
    width: 7px;

}


QSpinBox::up-arrow,
QDoubleSpinBox::up-arrow,
QDateTimeEdit::up-arrow

{
    image: url(:/icons/icons/arrow-up.png);
    width: 7px;

}


</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>252</width>
     <height>491</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Semibold</family>
    </font>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Segoe UI Semibold</family>
       </font>
      </property>
      <property name="text">
       <string>&lt;h2&gt;Проценты по вкладу&lt;/h2&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_usd_4">
            <property name="font">
             <font>
              <family>Segoe UI Semibold</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Условия 1: </string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="spin_p_1">
            <property name="minimumSize">
             <size>
              <width>71</width>
              <height>41</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>71</width>
              <height>41</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Segoe UI Semibold</family>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="minimum">
             <double>1.000000000000000</double>
            </property>
            <property name="maximum">
             <double>10.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_usd_8">
            <property name="minimumSize">
             <size>
              <width>42</width>
              <height>41</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>42</width>
              <height>41</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Segoe UI Semibold</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>%</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignHCenter|Qt::AlignTop</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_usd_5">
            <property name="font">
             <font>
              <family>Segoe UI Semibold</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Условия 2: </string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="spin_p_2">
            <property name="minimumSize">
             <size>
              <width>71</width>
              <height>41</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>71</width>
              <height>41</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Segoe UI Semibold</family>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="minimum">
             <double>1.000000000000000</double>
            </property>
            <property name="maximum">
             <double>10.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="label_usd_9">
            <property name="minimumSize">
             <size>
              <width>42</width>
              <height>41</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>42</width>
              <height>41</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Segoe UI Semibold</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>%</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignHCenter|Qt::AlignTop</set>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_usd_6">
            <property name="font">
             <font>
              <family>Segoe UI Semibold</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Условия 3: </string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QDoubleSpinBox" name="spin_p_3">
            <property name="minimumSize">
             <size>
              <width>71</width>
              <height>41</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>71</width>
              <height>41</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Segoe UI Semibold</family>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="minimum">
             <double>1.000000000000000</double>
            </property>
            <property name="maximum">
             <double>10.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLabel" name="label_usd_10">
            <property name="font">
             <font>
              <family>Segoe UI Semibold</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>%</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignHCenter|Qt::AlignTop</set>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_usd_7">
            <property name="minimumSize">
             <size>
              <width>117</width>
              <height>41</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Segoe UI Semibold</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Условия 4: </string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QDoubleSpinBox" name="spin_p_4">
            <property name="minimumSize">
             <size>
              <width>71</width>
              <height>41</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>71</width>
              <height>41</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Segoe UI Semibold</family>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="minimum">
             <double>1.000000000000000</double>
            </property>
            <property name="maximum">
             <double>10.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QLabel" name="label_usd_11">
            <property name="font">
             <font>
              <family>Segoe UI Semibold</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>%</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignHCenter|Qt::AlignTop</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>Segoe UI Semibold</family>
       </font>
      </property>
      <property name="text">
       <string>&lt;h2&gt;Курсы валют&lt;/h2&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label_usd_2">
            <property name="font">
             <font>
              <family>Segoe UI Semibold</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>1 RUB  = </string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="spin_usd">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>71</width>
              <height>41</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>71</width>
              <height>41</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Segoe UI Semibold</family>
              <pointsize>8</pointsize>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="minimum">
             <double>1.000000000000000</double>
            </property>
            <property name="maximum">
             <double>150.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_usd">
            <property name="font">
             <font>
              <family>Segoe UI Semibold</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>USD</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_usd_3">
            <property name="font">
             <font>
              <family>Segoe UI Semibold</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>1 RUB  = </string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="spin_eur">
            <property name="minimumSize">
             <size>
              <width>71</width>
              <height>41</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>71</width>
              <height>41</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Segoe UI Semibold</family>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="minimum">
             <double>1.000000000000000</double>
            </property>
            <property name="maximum">
             <double>150.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="label_eur">
            <property name="font">
             <font>
              <family>Segoe UI Semibold</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>EUR</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Maximum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QPushButton" name="button_save">
        <property name="font">
         <font>
          <family>Segoe UI Semibold</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Сохранить</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_discharge">
        <property name="font">
         <font>
          <family>Segoe UI Semibold</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Сбросить</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
